src/FinSearch.php

```diff
@@ -50,7 +50,7 @@ public function uninstall(UninstallContext $uninstallContext): void
         parent::uninstall($uninstallContext);
     }
 
-    private function getServiceXml(): string
+    private function getServiceXml($container): string
```

Comment:
1. This is not used.
1. It does not have a type hint.

Reply:
yes this was due to the version changes I was doing. It is not needed will remove it.